fullstack-cert/python-projects/game-character-stats/user-stories.md

```diff
@@ -0,0 +1,21 @@
+1. Create a class named `GameCharacter` that represents a game character. The class should manage stats such as name, health, mana, and level. In the `__init__` method:
+    - Accept a parameter `name` and initialize the private attribute `__name` with it.
+    - Initialize `__health` to `100`, `__mana` to `50`, and `__level` to `1`.
+    - Use double underscores (`__`) to ensure that all attributes are private.
```

Comment:
```suggestion
1. Create a class named `GameCharacter` that represents a game character and manages character stats.
1. When instantiated, a new `GameCharacter` object should have the following private attributes:
    - `__name` set to the string given at the moment of the instantiation.
    - `__health` set to `100`.
    - `__mana` set to `50`.
    - `__level` set to `1`.
```
I would not mention the `__init__` method because they should know how to do it at this point.